File: src/problem_1/problem_1.c
#include <stdio.h>

/*
 * Solution to problem 1
 *
 * https://projecteuler.net/problem=1
 */


int main(void)
{
    int solution = run();
    printf("%i", solution);
}

int run()
{
    int sum = 0;
    int ii;
    for (ii = 0; ii < 1000; ii++) {
        if (ii % 3 == 0 || ii % 5 == 0)
            sum += ii;
    }
    return sum;
}

File: src/problem_1/problem_1.py
# Solution to problem 1
#
# https://projecteuler.net/problem=1


def run():
    s = 0
    for x in range(0, 1000):
        if x % 3 == 0 or x % 5 == 0:
            s += x
    return s

print(run())